tests/migration_test.go

```diff
@@ -42,7 +42,7 @@ import (
 	"kubevirt.io/kubevirt/tests/clientcmd"
 	"kubevirt.io/kubevirt/tests/framework/checks"
 	"kubevirt.io/kubevirt/tests/libnode"
-	"kubevirt.io/kubevirt/tests/util"
+	util "kubevirt.io/kubevirt/tests/util"
```

Comment:
Is this change intentional/helpful?

Reply:
Yes, it's used here:
```
vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
```
Go (something that analyzes the code) added that line automatically as `util2 "kubevirt.io/kubevirt/tests/util"` that was used here:
```
vmiPod, err := getRunningPodByVirtualMachineInstance(vmi, util2.NamespaceTestDefault)
```
so I just called it `util`. I see that it can work without `util` and it's not needed (it was used by other code). I'd like to run `make functest` again.

For now please don't let CI run to save resources. I'd like to re-run it locally again on Sunday (couldn't run now in dockerized env because it doesn't work for some reason).

Reply:
Removed `util` from there. Ran tests locally successfully. Please re-review it :)